controllers/administration.js

```diff
@@ -1495,13 +1495,16 @@ router.get(
 						? consent.parentConsents[0]
 						: {};
 				}
+				const canDeleteStudent = res.locals.currentUser.permissions.includes('STUDENT_DELETE');
 				const hidePwChangeButton = !account;
```

Comment:
I think a better way is to use something like on line 950:
const hasEditPermission = permissionsHelper.userHasPermission(currentUser, 'TEACHER_EDIT');
this allows us to alter latter the function and adding more functionality for example (is what I was trying to do in another ticket)